woocommerce/woocommerce-gutenberg-products-block
# Review blocks: fix attribute notices not appearing

This PR makes the attributes notices to appear again. It also removes some unused CSS classes and creates common classes for `.wc-block-reviews__notice` and `.wc-block-reviews__selection` so we don't need to repeat them for each block.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/63340006-661fd500-c346-11e9-8f06-d9a1938edd38.png)

### How to test the changes in this Pull Request:

1. In _WooCommerce_ > _Settings_ > _Products_ > _General_ > _Product ratings_ uncheck _Enable star rating on reviews_.
2. In _Settings_ > _Discussion_ > _Avatar Display_ uncheck _Show Avatars_.
3. Create a post with a _Reviews by Product_ block.
4. Enable all settings and verify proper notices appear (see screenshot above).
